persistence/src/main/scala/com/evolutiongaming/akkaeffect/persistence/EventSourcedActorOf.scala

```diff
@@ -1,44 +1,39 @@
 package com.evolutiongaming.akkaeffect.persistence
 
+import java.time.Instant
+
 import akka.actor.Actor
 import akka.persistence.SnapshotSelectionCriteria
-import cats.effect.Async
-import cats.effect.Ref
-import cats.effect.Resource
+import cats.Monad
 import cats.effect.implicits.effectResourceOps
+import cats.effect.{Async, Ref, Resource}
 import cats.syntax.all._
 import com.evolutiongaming.akkaeffect._
-import com.evolutiongaming.catshelper.LogOf
-import com.evolutiongaming.catshelper.ToFuture
-
-import java.time.Instant
-import scala.reflect.ClassTag
+import com.evolutiongaming.akkaeffect.persistence.SeqNr
+import com.evolutiongaming.catshelper.{LogOf, ToFuture}
 
 object EventSourcedActorOf {
 
+  // format: off
   /** Describes lifecycle of entity with regards to event sourcing & PersistentActor Lifecycle phases:
+    * 1. RecoveryStarted: we have id in place and can decide whether we should continue with recovery 
+    * 2. Recovering : reading snapshot and replaying events 
+    * 3. Receiving : receiving commands and potentially storing events & snapshots 
+    * 4. Termination : triggers all release hooks of allocated resources within previous phases
     *
-    *   1. RecoveryStarted: we have id in place and can decide whether we should continue with recovery 2. Recovering : reading snapshot and
-    *      replaying events 3. Receiving : receiving commands and potentially storing events & snapshots 4. Termination : triggers all
-    *      release hooks of allocated resources within previous phases
-    *
-    * Types, describing each phase, are (simplified) a functions from data (available on the current phase) to next phase: RecoveryStarted
-    * -> Recovering -> Receiving. Termination phase described via aggregation of [[Resource]] release callbacks. Please refer to phase types
-    * for more details.
+    * Types, describing each phase, are (simplified) a functions from data (available on the current phase) to next phase: 
+    * RecoveryStarted -> Recovering -> Receiving. 
+    * Termination phase described via aggregation of [[Resource]] release callbacks. Please refer to phase types for more details.
     *
-    * @tparam S
-    *   snapshot
-    * @tparam E
-    *   event
-    * @tparam C
-    *   command
+    * @tparam S snapshot
+    * @tparam E event
+    * @tparam C command
```

Comment:
the original code is as ScalaDoc expects it to be formatted - with adjusted list setup, there will be no need for `// format: off` and `: on` extras

Reply:
how could I adjust scalafmt for having correct scaladoc formatting? I would prefer to keep the doc as is, but preferably without `format: off/on`